### DISCUSSION TITLE
How can i get other data and put in into theme

### DISCUSSION THREAD

**Discussion by dobybot-git** [mathoudebine/turing-smart-screen-python#290] [Q&A] [answered]
Hi, 

I have already made a Theme with signal from CPU / MEM / NETWORK but can't find a way to put data from other sources into it.

I want to put other data like Docker's information or message from mqtt. 

Anyone can help?

**Comment by mathoudebine** [comment]
Hello, for now custom data sources are not supported by the System Monitor program. It could be useful to add a way for this program to get and display them.
Until it is implemented, you can look at `simple-program.py` to get and display custom data on the screen from scratch, but with this solution you would lose all system monitor features like themes, hardware stats...

**Comment by mathoudebine** [comment, marked as answer]
Hello @dobybot-git,
This is now implemented in the [Release 3.3.3](https://github.com/mathoudebine/turing-smart-screen-python/releases/tag/3.3.3)!
Please see the release note to get started!
